.github/workflows/connector-test-command.yml

```diff
@@ -0,0 +1,333 @@
+name: Connector CI (Beta)
+
+on:
+  pull_request:
+    types: [opened, synchronize, reopened]
+
+  # Available as a reusable workflow
+  # (https://docs.github.com/en/actions/sharing-automations/reusing-workflows)
+  workflow_call:
+    inputs:
+
+      connector_name:
+        type: string
+        required: false
+        description: >
+          The name of the connector(s) to test.
+          To test multiple connectors, use a newline-delimited
+          or comma-separated list of connector names.
+
+          If not provided, the workflow will try to infer the
+          connector name from the PR title.
+
+      repo:
+        type: string
+        required: false
+        description: "The repository name"
+      gitref:
+        type: string
+        required: false
+        description: "The git reference (branch or tag)"
+      comment-id:
+        type: number
+        required: false
+        description: "The ID of the comment triggering the workflow"
+      pr:
+        type: number
+        required: false
+        description: "The pull request number, if applicable"
+
+  # Available as a slash command and workflow dispatch
+  workflow_dispatch:
+    inputs:
+
+      # Custom inputs for the connector test command:
+      connector_name:
+        required: false
+        description: |
+          The name of the connector(s) to test.
+          To test multiple connectors, use a newline-delimited
+          or comma-separated list of connector names.
+
+          If not provided, the workflow will try to infer the
+          connector name from the PR title.
+
+      # Static args from the slash command dispatch:
+      repo:
+        required: false
+        description: "The repository name"
+      gitref:
+        required: false
+        description: "The git reference (branch or tag)"
+      comment-id:
+        required: false
+        description: "The ID of the comment triggering the workflow"
+      pr:
+        required: false
+        description: "The pull request number, if applicable"
+
+jobs:
+  generate-matrix:
+    name: Generate Connector Matrix
+    runs-on: ubuntu-24.04
```

Comment:
Just curios, if there is anything smaller to run on?

Reply:
@bazarnov - This is already the smallest/default container size, I believe. In other places we use custom 'large' images, but that is mostly because we were running multiple connectors' tests on the same container.

Reply:
Confirmed: https://docs.github.com/en/actions/using-github-hosted-runners/using-github-hosted-runners/about-github-hosted-runners#standard-github-hosted-runners-for-public-repositories